.github/workflows/delivery/homebrew/pack.rb

```diff
@@ -5,7 +5,10 @@
 class Pack < Formula
   desc "A CLI for building apps using Cloud Native Buildpacks"
   homepage "https://github.com/buildpacks/pack"
-  if OS.mac?
+  if OS.mac? && Hardware::CPU.arm?
```

Comment:
This was based off how openJDK added Mac ARM support: https://github.com/Homebrew/homebrew-core/pull/68946